final/problem-def-ana/problem-def.tex

```diff
@@ -1,10 +1,8 @@
 \section{Problem Definition}
 \label{sec:problem-definition}
 
-The previous sections provided the necessary background knowledge of
-the problem domain. Now that the background knowledge has been
-explained, this section will discuss the problem definition of the
-client.
+Now that the background knowledge has been explained, this section discusses the
+problem definition of the client.
```

Comment:
Hendrik's feedback mentions the following:

> The structure of scientific text is usually as follows:
1. General statement, problem, question.
2. Go into details about (1)
3. General conclusion, summary, solution.
This structure appears on every level, so from chapters to paragraphs. Thinking about (1) and (3) usually helps to determine if all the details in (2) actually belong there.

So while I agree there is some double text, I do believe this is the way to go.